deployment/copilot/README.md

```diff
@@ -186,5 +190,16 @@ Official documentation on Addons can be found [here](https://aws.github.io/copil
 
 ### Useful Commands
 
-`copilot app show`: Provides details on the current app \
-`copilot svc show`: Provides details on a particular service
+`copilot app show` - Provides details on the current app \
+`copilot svc show` - Provides details on a particular service
+
+### Removing deloyed resources from AWS
+
+To remove the resources installed from the steps above, follow these instructions:
+1. Access the AWS Console and delete all S3 objects and content within the "StagingBucket".
+2. Execute the following commands:
```

Comment:
I would add more details here: use this command if you want to delete all resources for a given env/stage. The following command will delete all remaining app level stacks in all environments to return to a clean slate (except for the bootstrap stack if it was required)